src/cls/refcount/cls_refcount.cc

```diff
@@ -13,6 +13,29 @@ CLS_NAME(refcount)
 
 
 #define REFCOUNT_ATTR "refcount"
+#define CHUNK_REFCOUNT_ATTR "chunk_refcount"
+
+struct chunk_obj_refcount {
+  map<chunk_refcount, bool> refs;
+  set<chunk_refcount> retired_refs;
```

Comment:
it doesn't seem like retired_refs is serving any  purpose?

Reply:
This is my mistake. I will fix this.